Components/Queues/Drivers/StompQueueDriver.php

```diff
@@ -365,7 +393,7 @@ private function unescape($string)
     }
 
     /** {@inheritdoc} */
-    public function ack()
+    public function ack(int $messageId = null)
```

Comment:
I think that instead of passing the messageId, we should pass the message object itself and then the driver decides what is needed to ack or nack

Reply:
This was changed to get the QueueMessage object.